pkg/aggregator/aggregator.go

```diff
@@ -168,6 +168,10 @@ var (
 	tlmChecksContextsBytesByMtype = telemetry.NewGauge("aggregator", "checks_contexts_bytes_by_mtype",
 		[]string{"shard", "metric_type", tags.BytesKindTelemetryKey}, "Estimated count of bytes taken by contexts in the check aggregator, by metric type")
 
+	agentRunning = telemetry.NewGauge("runtime", "running",
+		[]string{}, "Indicates if the agent is running",
+	)
+
```

Comment:
The values never change and do not explain anything about the aggregator itself. Can we please implement these new metrics directly in the agenttelemetry component, as to not to pollute aggregator with this unrelated function?

(Yes, aggregator reports similar metrics to the user org, because it was a natural place to inject metrics without a clear user configured source. No, this isn't a good reason to place new metrics here as well.)

Reply:
My logic was agenttelemetry is the component responsible for COAT in the agent, so I think it is reasonable to put COAT-specific things there. But I don't have a strong opinion, we could do it at `cmd` level as well where we init the components, since these metrics are about the entire agent. But I do strongly believe that topical components, like aggregator, should only report telemetry about the component itself, and the entire agent.

Reply:
I do not think that COAT should be emitting this metric either. No can it. I looks like one use case it to detect then Agent started but never reach running state When and where we know for sure that the Agent is running? I do not know, it is a good question as well if it aggregators responsibility. But I think that @louis-cqrl is only a messenger here because that decision had been made a while back in the form of updating a different metric with the same semantic intent (there is a limitation why COAT cannot reuse existing metric otherwise it would not be necessary).